operations/host_setup.go

```diff
@@ -44,7 +44,7 @@ func hostSetup() cli.Command {
 func runSetupScript(ctx context.Context, wd string, setupAsSudo bool) error {
 	grip.Warning(os.MkdirAll(wd, 0777))
 
-	if _, err := os.Stat(evergreen.SetupScriptName); os.IsNotExist(err) {
+	if err := os.Rename(evergreen.TempSetupScriptName, evergreen.SetupScriptName); os.IsNotExist(err) {
```

Comment:
I don't have a particularly good argument for it, except that it would require _only_ agent changes, but what if the rename goes in the other direction, from the original name to a temp file?

Reply:
I changed this to only rename to the temp file name in `evergreen host setup` so setupHostJob won't have to change.